### PR TITLE
Rust/may-minihttp add may-minihttp framework

### PR DESCRIPTION
**Pull request by Xudong-Huang**
add may-minihttp framework, currently not include database related test.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -181,6 +181,7 @@ env:
     # - "TESTDIR=Rust/iron"
     # - "TESTDIR=Rust/nickel"
     # - "TESTDIR=Rust/hyper"
+    # - "TESTDIR=Rust/may-minihttp"
```
Comment (zloster) [inline .travis.yml]: This line should NOT be commented. There is a revert of some accidental merge of the Docker branch into master (https://github.com/TechEmpower/FrameworkBenchmarks/pull/3357) several commits later.
@Xudong-Huang @nbrady-techempower

Comment (nbrady-techempower) [reply .travis.yml]: This is in #3358 now

Comment (Xudong-Huang) [reply .travis.yml]: Hope someone would fix it, I just see all rust related test is disabled currently, I think there is a reason.

Comment (nbrady-techempower) [reply .travis.yml]: @Xudong-Huang it is fixed in the docker branch. We're moving toward docker for all test implementations.

**Pull request merged**